mysite/polls/tests.py

```diff
@@ -107,6 +108,88 @@ def test_past_question(self):
         self.assertContains(response, past_question.question_text)
 
 
+class QuestionVoteViewTests(TestCase):
```

Comment:
In general when a file is growing too much like this one, and the purpose of each class could be isolated, it is better to separate it into different files and greater each test class individually in its own file.
that way we are reducing complexity to the tests and the process of finding a specific test is not so cumbersome.

Reply:
Yes, that would work. **The polls/tests.py** is still small but the **api/tests.py** is 1500+ lines. I'll break it down and push the changes to this branch as soon as possible.